config/routes.rb

```diff
@@ -115,7 +115,6 @@
       get '/login/two_factor/piv_cac' => 'two_factor_authentication/piv_cac_verification#show'
       get '/login/two_factor/piv_cac/present_piv_cac' => 'two_factor_authentication/piv_cac_verification#redirect_to_piv_cac_service'
       get '/login/two_factor/webauthn' => 'two_factor_authentication/webauthn_verification#show'
-      get '/login/two_factor/webauthn_error' => 'two_factor_authentication/webauthn_verification#error'
```

Comment:
After checking logs, there's _some_ traffic to this page still, but reasonably low overall.

Also worth noting the actual experience for someone on an unsupported browser should be marginally improved by these changes, since they'll still see an error message when trying to authenticate. Unlike before, they'll now have the option to select another MFA method on the unsupported device, which was not possible previously.